### PR TITLE
Newsletters: add 215 (2022-08-31)

### PR DESCRIPTION
**Pull request by harding**
- [x] Update releases/RCs and add topic links @harding 
- [x] Stack Exchange section @bitschmidty

### CODE REVIEW

File: _posts/en/newsletters/2022-08-31-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+label export format and includes our regular sections with summaries of
+recent questions and answers from the Bitcoin StackExchange, a list of
+new software releases and release candidates, and descriptions of
+notable changes to popular Bitcoin infrastructure software.
+
+## News
+
+- **Wallet label export format:** Craig Raw [posted][raw
+  interchange] to the Bitcoin-Dev mailing list a proposed BIP to
+  standarize the format wallets use to export labels for their addresses
```
Comment (bitschmidty) [inline _posts/en/newsletters/2022-08-31-newsletter.md:20]:
```suggestion
  standardize the format wallets use to export labels for their addresses
```

File: _posts/en/newsletters/2022-08-31-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  messages] with the ability to send *reply paths*, and to decode them
+  when received.  The onion message protocol doesn't require a node
+  which relays an onion message to track any information about that
+  message after relay, so it can't automatically send a reply back
+  along the path the original message took.  That means a node which
+  wants a reply to its onion message needs to provide hints to the
+  receiver about what path to use for sending a reply.
+
+- [HWI #627][] adds support for [P2TR][topic taproot] keypath spends using
+  the Bitbox02 hardware signing device.
```
Comment (bitschmidty) [inline _posts/en/newsletters/2022-08-31-newsletter.md:132]:
```suggestion
  the BitBox02 hardware signing device.
```

File: _posts/en/newsletters/2022-08-31-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Why isn't it possible to add an OP_RETURN (or some arbitrary script) inside a taproot script path with a descriptor?]({{bse}}114948)
+  Antoine Poinsot explains that [script descriptors][topic descriptors] are
+  currently being extended to use [miniscript][topic miniscript] in Bitcoin
+  Core and expected in the Bitcoin Core 24.0 release. While initial miniscript
+  features will only include segwit v0 support,
+  eventually support for [tapscript][topic tapscript] could be added and
+  discussions about [partial descriptors][Bitcoin Core #24114] are currently underway.
```
Comment (Xekyo) [inline _posts/en/newsletters/2022-08-31-newsletter.md:58]: TBH, this sounds like an interesting topic, but I have read the answer multiple times now, and I'm still not sure I understand what it's saying about OP_RETURN in scriptpaths. I'm gonna see if I can try to clarify it tomorrow in the office.

File: _posts/en/newsletters/2022-08-31-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+
+- [Why does Bitcoin Core rebroadcast transactions?]({{bse}}114973)
+  Amir reza Riahi wonders why the Bitcoin Core wallet rebroadcasts transactions
+  and why there is a delay. Pieter Wuille points out the P2P network's lack of
+  transaction propogation guarantee as the reason for rebroadcasting being
```
Comment (Xekyo) [inline _posts/en/newsletters/2022-08-31-newsletter.md:63]:
```suggestion
  transaction propagation guarantee as the reason for rebroadcasting being
```

**Comment by harding** [comment]
Made suggested edits, updated releases/RCs, added topic links, and reviewed @bitschmidty SE section.  Thanks @bitschmidty and @Xekyo!

File: _posts/en/newsletters/2022-08-31-newsletter.md (hunk 3 above)
Comment (bitschmidty) [reply _posts/en/newsletters/2022-08-31-newsletter.md]: Ill leave as is for the newsletter for today. Let me know if there is feedback from your discussions.

**Comment by bitschmidty** [comment]
Squashed and merged, thank you @harding and @Xekyo !

File: _posts/en/newsletters/2022-08-31-newsletter.md (hunk 3 above)
Comment (Xekyo) [reply _posts/en/newsletters/2022-08-31-newsletter.md]: Yeah, it's fine in the newsletter, I just found the part about why it doesn't make a lot of sense to write an `OP_RETURN` into the script tree easy to follow. I wrote my own answer today: https://bitcoin.stackexchange.com/a/115100/5406

**Pull request merged**